lib/mauth/fake/rack.rb

```diff
@@ -14,30 +16,32 @@ module Rack
     # (rather than switching to this fake one), as all this does is add those keys to the request env.
     class RequestAuthenticationFaker < MAuth::Rack::RequestAuthenticator
       class << self
-        def is_authentic?
+        def is_authentic? # rubocop:disable Naming/PredicateName
           @is_authentic.nil? ? true : @is_authentic
         end
 
-        def authentic=(is_auth = true)
+        def authentic=(is_auth = true) # rubocop:disable Style/OptionalBooleanParameter
           @is_authentic = is_auth
         end
       end
 
       def call(env)
         retval = if should_authenticate?(env)
-          mauth_request = MAuth::Rack::Request.new(env)
-          env['mauth.protocol_version'] = mauth_request.protocol_version
+                   mauth_request = MAuth::Rack::Request.new(env)
+                   env['mauth.protocol_version'] = mauth_request.protocol_version
 
-          if self.class.is_authentic?
-            @app.call(env.merge!('mauth.app_uuid' => mauth_request.signature_app_uuid, 'mauth.authentic' => true))
-          else
-            response_for_inauthentic_request(env)
-          end
-        else
-          @app.call(env)
-        end
+                   if self.class.is_authentic?
+                     @app.call(env.merge!('mauth.app_uuid' => mauth_request.signature_app_uuid,
+                       'mauth.authentic' => true))
+                   else
+                     response_for_inauthentic_request(env)
+                   end
+                 else
+                   @app.call(env)
+                 end
```

Comment:
is that wide indent our favorite style ? 

I would vote for keeping it tight and use the rules described here

https://stackoverflow.com/questions/51892642/rubocop-indentation-configuration

Reply:
you can start discussion in https://github.com/mdsol/rubocop-mdsol